docs/source/Support/bskReleaseNotes.rst

```diff
@@ -28,7 +28,8 @@ Version |release|
 -----------------
 - Updated Linux and Windows CI builds to use ``swig`` 4.2.1
 - Updated CI scripts to run on latest macOS and no longer use Ubuntu 20.04
-
+- Updated ``makeDraftModule.py`` to remove redundant comments and implementation of the destructor,
```

Comment:
make a hyperlink to `makeDraftModule.py` documentation using :ref:`makeDraftModule`